### PR TITLE
Work for 1.1.4

### PR DESCRIPTION
**Pull request by StrangeZak**

### CODE REVIEW

**Comment by StrangeZak** [comment]
@bvisness Not sure whats wrong with this. Errors are not making much sense

**Comment by StrangeZak** [comment]
@DanielGibson This should fix your issue.

**Comment by StrangeZak** [comment]
Okay.. Im confused now why doesn't it know what Left and Right is. This compiles fine in my project

**Comment by StrangeZak** [comment]
Woohoo okay that was harder than it should've been. @bvisness take a look so we can merge this in!

File: HandmadeMath.h [hunk 1]
```diff
@@ -1336,9 +1360,24 @@ HMM_SubtractMat4(hmm_mat4 Left, hmm_mat4 Right)
     return (Result);
 }
 
-HINLINE hmm_mat4
+#ifndef HANDMADE_MATH_NO_SSE
+inline __m128
```
Comment (bvisness) [inline HandmadeMath.h]: @StrangeZak `HINLINE`?

File: HandmadeMath.h [hunk 2]
```diff
@@ -1336,9 +1360,24 @@ HMM_SubtractMat4(hmm_mat4 Left, hmm_mat4 Right)
...
+    Result = _mm_mul_ps(_mm_shuffle_ps(Left, Left, 0x00), Right.Rows[0]);
+    Result = _mm_add_ps(Result, _mm_mul_ps(_mm_shuffle_ps(Left, Left, 0x55), Right.Rows[1]));
+    Result = _mm_add_ps(Result, _mm_mul_ps(_mm_shuffle_ps(Left, Left, 0xaa), Right.Rows[2]));
+    Result = _mm_add_ps(Result, _mm_mul_ps(_mm_shuffle_ps(Left, Left, 0xff), Right.Rows[3]));
+
+    return Result;
+}
+#endif
+
+hmm_mat4
```
Comment (bvisness) [inline HandmadeMath.h]: @StrangeZak You removed the `HINLINE` here.

File: HandmadeMath.h [hunk 3]
```diff
@@ -480,6 +500,10 @@ HMMDEF hmm_vec2 HMM_SubtractVec2(hmm_vec2 Left, hmm_vec2 Right);
 HMMDEF hmm_vec3 HMM_SubtractVec3(hmm_vec3 Left, hmm_vec3 Right);
 HMMDEF hmm_vec4 HMM_SubtractVec4(hmm_vec4 Left, hmm_vec4 Right);
 
+#ifndef HANDMADE_MATH_NO_SSE
+HMMDEF __m128 HMM_LinearCombineSSE(__m128 Left, hmm_mat4 Right);
+#endif
+
```
Comment (bvisness) [inline HandmadeMath.h]: @StrangeZak We need a test for `HMM_LinearCombineSSE`.

File: HandmadeMath.h (hunk 2 above)
Comment (StrangeZak) [inline HandmadeMath.h]: Fixed

File: HandmadeMath.h (hunk 1 above)
Comment (StrangeZak) [inline HandmadeMath.h]: Fixed

File: HandmadeMath.h [hunk 4]
```diff
@@ -237,7 +243,7 @@ extern "C"
 
 #if !defined(HMM_SINF) || !defined(HMM_COSF) || !defined(HMM_TANF) || \
     !defined(HMM_EXPF) || !defined(HMM_LOGF) || !defined(HMM_ACOSF) || \
-    !defined(HMM_ATANF)|| !defined(HMM_ATAN2F)
+    !defined(HMM_ATANF)|| !defined(HMM_ATAN2F) || !defined(HMM_SQRTF)
```
Comment (StrangeZak) [inline HandmadeMath.h]: Fixed

**Comment by StrangeZak** [comment]
Yep, will do it soon

**Comment by StrangeZak** [comment]
Closing this out changed some stuff on my internal copy and gonna implement the changes into master instead of handling conflicts here.

**Pull request closed without merging**